sunpy/map/map_factory.py

```diff
@@ -79,6 +77,20 @@ class MapFactory(BasicRegistrationFactory):
 
     >>> mymap = sunpy.map.Map(data, header)   # doctest: +SKIP
```

Comment:
This is definitely the easiest way to demonstrate this functionality but I wonder if it's a little confusing. I will have a think if there is a better way to do it

Reply:
Confusing now?

Reply:
I mean its not the way you *should* load a FITS file. Its if you have a WCS object from some other source. I am really not sure there is a better way though.

Reply:
OH, I get you now. I mean we could construct one in the example instead?